Automattic/wp-calypso
# Patterns API: Remove old block pattern files and code.

#### Changes proposed in this Pull Request

* Remove all of the old block patterns code from the patterns toolkit. All of the patterns are loaded from the API rather than from files in the editing toolkit plugin. These files are no longer loaded so can be safely removed.

#### Testing instructions



* Check out the PR, and sync to your sandbox using `yarn dev --sync`
* Load a sandboxed site and confirm you still see these patterns:

<img width="159" alt="Screen Shot 2020-11-10 at 1 54 01 PM" src="https://user-images.githubusercontent.com/1464705/98738197-32bad300-235c-11eb-8dff-cec10f3deb75.png">
